feature-libs/organization/account-summary/root/http-interceptors/blob-error.interceptor.ts

```diff
@@ -13,14 +13,18 @@ import {
 } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { FileReaderService } from '@spartacus/storefront';
+import { WindowRef } from '@spartacus/core';
 import { Observable, throwError } from 'rxjs';
 import { catchError, switchMap } from 'rxjs/operators';
 
 @Injectable({
   providedIn: 'root',
 })
 export class BlobErrorInterceptor implements HttpInterceptor {
-  constructor(private fileReaderService: FileReaderService) {}
+  constructor(
+    protected fileReaderService: FileReaderService,
+    protected windowRef: WindowRef
```

Comment:
Let's use the `inject` function instead of the new parameter in the constructor to reduce the amount of breaking changes and to be in line with the agreement we introduced in the Spartacus project.

Reply:
This wasn't part of public API, I don't think we need to use inject

Reply:
But it's going to be a part of public API and it's hard to predict whether we'll remove this injection in the future or not. That said, actually I propose, completely removing the `constructor` and using the `inject` function in both cases :)